modules/serverless-autoscaling-developer-maxscale.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * serverless/develop/serverless-autoscaling-developer.adoc
+
+:_content-type: REFERENCE
+[id="serverless-autoscaling-developer-maxscale_context"]
+= Maximum scale bounds
+
+The maximum number of replicas that can serve an application is determined by the `maxScale` annotation. If the `maxScale` annotation is not set, there is no upper limit for the number of replicas created.
+
+.Example service spec with `maxScale` spec
```

Comment:
You have spec in here twice. Should the last "spec" be something else, like "annotation"?